src/Microsoft.IdentityModel.Tokens/Base64UrlEncoding.cs

```diff
@@ -3,48 +3,19 @@
 
 using System;
 using System.Buffers;
+using System.Buffers.Text;
 using Microsoft.IdentityModel.Logging;
 
 namespace Microsoft.IdentityModel.Tokens
 {
-    /// <summary>
-    /// Base64 encode/decode implementation for as per https://tools.ietf.org/html/rfc4648#section-5.
-    /// Uses ArrayPool[T] to minimize memory usage.
-    /// </summary>
     internal static class Base64UrlEncoding
     {
-        private const uint IntA = 'A';
-        private const uint IntZ = 'Z';
-        private const uint Inta = 'a';
-        private const uint Intz = 'z';
-        private const uint Int0 = '0';
-        private const uint Int9 = '9';
-        private const uint IntEq = '=';
-        private const uint IntPlus = '+';
-        private const uint IntMinus = '-';
-        private const uint IntSlash = '/';
-        private const uint IntUnderscore = '_';
-
-        private static readonly char[] Base64Table =
-        {
-            'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O',
-            'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd',
-            'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's',
-            't', 'u', 'v', 'w', 'x', 'y', 'z', '0', '1', '2', '3', '4', '5', '6', '7',
-            '8', '9', '-', '_',
-        };
-
         /// <summary>
         /// Decodes a base64url encoded string into a byte array.
         /// </summary>
         /// <param name="inputString">The base64url encoded string to decode.</param>
         /// <returns>The decoded bytes.</returns>
-        public static byte[] Decode(string inputString)
-        {
-            _ = inputString ?? throw LogHelper.LogArgumentNullException(nameof(inputString));
-
-            return Decode(inputString, 0, inputString.Length);
-        }
+        public static byte[] Decode(string inputString) => Decode(inputString, 0, inputString.Length);
```

Comment:
Are there unit tests for these:
`Decode(string inputString)`
`Decode(string input, int offset, int length)`